packages/host/tests/acceptance/interact-submode-test.gts

```diff
@@ -1169,6 +1170,10 @@ module('Acceptance | interact submode tests', function (hooks) {
         `[data-test-card-catalog-create-new-button="${testRealm3URL}"]`,
       );
       await click(`[data-test-card-catalog-go-button]`);
+      await fillIn(
+        `[data-test-stack-card-index="1"] [data-test-field="name"] input`,
+        'Mango',
+      );
```

Comment:
this is the cross realm optimistic card creation scenario. flesh this test out even futher to show that the consuming realm was updated to point to new link